client/allocrunnerv2/taskrunner/task_runner.go

```diff
@@ -95,8 +96,9 @@ type TaskRunner struct {
 	// driver is the driver for the task.
 	driver driver.Driver
 
-	handle     driver.DriverHandle // the handle to the running driver
-	handleLock sync.Mutex
+	handle       driver.DriverHandle // the handle to the running driver
```

Comment:
Same comment

Reply:
comments start with field name

Reply:
Done! It's also worth noting that I think the weird little `handleResult` result proxy will be able to get dropped once DriverPlugins land as their Wait mechanism returns a result to every caller.